deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/PodsTable.tsx

```diff
@@ -36,14 +38,19 @@ const loader = async ({ request }: LoaderFunctionArgs): Promise<LoaderData> => {
   const order = getOrderFromSearchParams(searchParams);
 
   const kubernetesStatus = searchParams.get('kubernetes_state');
-
+  const hosts = searchParams.get('hosts')?.split(',') ?? [];
+  const clusters = searchParams.get('clusters')?.split(',') ?? [];
+  const filterIn: {
+    [key: string]: string[];
+  } = hosts.length ? { host_name: hosts, kubernetes_cluster_name: clusters } : {};
   const searchSearchNodeReq: SearchSearchNodeReq = {
     node_filter: {
       filters: {
         compare_filter: null,
         contains_filter: {
           filter_in: {
             active: [true],
+            ...filterIn,
```

Comment:
same

Reply:
removed local var and added inline conditional logic